student.rb

```diff
@@ -0,0 +1,11 @@
+require('./person')
+class Student < Person
+  def initialize(classroom, age, parent_permission, name)
```

Comment:
- [OPTIONAL]  As requested by the previous code reviewer, kindly set the default name to 'Unknown' and default parent_permission to true, something like so: `def initialize(age, classroom, name = "Unknown", parent_permission = true)  `